Users/service.js

```diff
@@ -1,51 +1,63 @@
 var response = require('../Objects/response.js');
 
-var createUser = function (json) {
-    // Creates a user
-
-    // TODO: Create user and remove console.log
-    console.log(json);
+var createUser = (json) => {
+	// Create Student
+	if (json.role == 'Student') {
+		var Student = new StudentModel({
+			approvalStatus: json.approvalStatus,
+			email: json.email,
+			firstName: json.firstName,
+			lastName: json.lastName,
+			points: json.points,
+			role: json.role,
+			studentId: json.studentId
+		});
+	// TODO: Write new Student to database
+	}
+
+	// Create Admin
+	else if (json.role == 'Admin') {
+		var Admin = new AdminModel({
+			email: json.email,
+			firstName: json.firstName,
+			lastName: json.lastName,
+			role: json.role
+		});
+	// TODO: Write new Admin to database
+	}
 
     return new response.ResponseObject(201, {"uid":"1", "url":"//www.google.com"});
 }
 
-var modifyUser = function (id, json) {
-    // Modifies user
-
-    // TODO: Send to modifyUserAsSelf or modifyUserAsAdmin
-    // TODO: Remove 
-    console.log(id, json);
+var modifyUser = (id, json) => {
+	// Modify user as self
+    if (json.requesterUid == 0) {
```

Comment:
Since the way in which authentication information will be passed hasn't been decided, I'd just leave checking whether it's an admin as a TODO.